polkadot-protocol/onchain-governance/overview.md

```diff
@@ -1,4 +1,326 @@
 ---
 title: Overview
 description: TODO
----
\ No newline at end of file
+---
+
+# Governance Overview
+
+Polkadot employs an advanced governance system designed to facilitate continuous, adaptable evolution directed by its community of stakeholders. 
+
+This approach ensures that those holding the majority of stake retain control over the network’s direction.
+
+At the core of Polkadot’s innovation are several novel mechanisms, including a flexible, on-chain state-transition function written in **WebAssembly** (a platform-agnostic language) and multiple on-chain voting methods, such as referenda and batch approval voting. 
```

Comment:
This paragraph does not make a lot of sense to me, I'd remove this part: "flexible, on-chain state-transition function written in **WebAssembly**", and just focus on the on-chain voting mechanism. WDYT of the following:

```suggestion
The Polkadot blockchain incorporates a powerful on-chain governance mechanism, empowering the community to actively participate in decision-making and shape the network's future through decentralized, on-chain processes.
```